src/main/java/mil/dds/anet/services/MartDictionaryService.java

```diff
@@ -0,0 +1,224 @@
+      task.put("guid", childTask.getUuid());
+      task.put("name", childTask.getShortName());
+      tasks.add(task);
+    }
+    return tasks;
+  }
+
+  private List<Map<String, Object>> exportCommandsToMartDictionary(
+      List<Organization> regionalCommands) {
+    final List<Map<String, Object>> commands = new ArrayList<>();
+    regionalCommands.sort(Comparator.comparing(Organization::getShortName));
+    for (Organization org : regionalCommands) {
+      final Map<String, Object> command = new LinkedHashMap<>();
+      final List<Map<String, String>> reportingTeams = new ArrayList<>();
+      command.put("guid", org.getUuid());
+      command.put("name", org.getShortName());
+      OrganizationSearchQuery organizationSearchQuery = new OrganizationSearchQuery();
+      organizationSearchQuery.setStatus(WithStatus.Status.ACTIVE);
+      List<Organization> rts = org.loadChildrenOrgs(graphQLContext, null).join();
+      rts.sort(Comparator.comparing(Organization::getShortName));
+      for (Organization rt : rts) {
+        final Map<String, String> reportingTeam = new LinkedHashMap<>();
+        reportingTeam.put("guid", rt.getUuid());
+        reportingTeam.put("name", rt.getShortName());
+        reportingTeams.add(reportingTeam);
+      }
+      command.put("reportingTeams", reportingTeams);
+      commands.add(command);
+    }
+    return commands;
+  }
+
+  private List<Map<String, Object>> exportMunicipalitiesToMartDictionary(Location municipalityGroup)
+      throws JsonProcessingException {
+    final List<Map<String, Object>> result = new ArrayList<>();
+    List<Location> municipalities =
+        new ArrayList<>(municipalityGroup.loadChildrenLocations(graphQLContext).join().stream()
+            .filter(m -> m.getStatus() == WithStatus.Status.ACTIVE).toList());
+    municipalities.sort(Comparator.comparing(Location::getName));
+    for (Location m : municipalities) {
+      final Map<String, Object> municipality = new LinkedHashMap<>();
+      final List<Map<String, Object>> locations = new ArrayList<>();
+
+      // Municipality fields
+      municipality.put("guid", m.getUuid());
+      addCustomFields(MART_DICT_MUNICIPALITY_CUSTOM_FIELDS, municipality, m);
+      List<Location> municipalityLocations = new ArrayList<>(m.loadChildrenLocations(graphQLContext)
+          .join().stream().filter(l -> l.getStatus() == WithStatus.Status.ACTIVE).toList());
+      municipalityLocations.sort(Comparator.comparing(Location::getName));
+      for (Location l : municipalityLocations) {
+        final Map<String, Object> location = new LinkedHashMap<>();
+        // Location fields
+        location.put("guid", l.getUuid());
+        addCustomFields(MART_DICT_LOCATION_CUSTOM_FIELDS, location, l);
+        locations.add(location);
+      }
+      municipality.put("locations", locations);
+      result.add(municipality);
+    }
+    return result;
+  }
+
+  private void addCustomFields(String dictionaryEntry, Map<String, Object> location,
```

Comment:
Method could be a little more robust:
```java
    final ObjectMapper objectMapper = new ObjectMapper();
    final String customFields = anetLocation.getCustomFields();
    if (!Utils.isEmptyOrNull(customFields)) {
      final JsonNode jsonNode = objectMapper.readTree(customFields);
      for (final String customField : getCustomFieldsFromDictionary(dictionaryEntry)) {
        final JsonNode customFieldNode = jsonNode.get(customField);
        if (customFieldNode != null) {
          location.put(customField, customFieldNode.asText());
        }
      }
    }
```
So as not to throw any errors if locations don't have custom fields defined.